core/src/desul/atomics/CUDA.hpp

```diff
@@ -453,6 +450,14 @@ namespace desul {
     return Impl::atomic_fetch_oper(Impl::AddOper<long, const long>(), dest, val, order, scope);
   }
   __device__ inline
+  unsigned long atomic_fetch_add(unsigned long* const dest, unsigned long val, MemoryOrderRelaxed order, MemoryScopeDevice scope) {
+    return Impl::atomic_fetch_oper(Impl::AddOper<unsigned long, const unsigned long>(), dest, val, order, scope);
+  }
+  __device__ inline
+  double atomic_fetch_add(double* const dest, double val, MemoryOrderRelaxed order, MemoryScopeDevice scope) {
+    return Impl::atomic_fetch_oper(Impl::AddOper<double, const double>(), dest, val, order, scope);
+  }
```

Comment:
I think this one (and the corresponding `atomic_fetch_add -> double`) might be only needed for Pascal

Reply:
Broke Clang build on Volta

Reply:
@crtrott @ndellingwood do we have a Clang+CUDA nightly build on Pascal?